### ISSUE TITLE
Ported to linux

### ISSUE THREAD

**Issue by florintanasa** [Tsukihime/OpenExpertSDR#1]
Hi,
I ported your work to linux and can be compiled with qt4 or qt5, also work compiled for windows.
You found the last source code at
https://copy.com/rDPM3sIHKou7AkVi

73, YO8TFF, Florin Tanasa

**Comment by Tsukihime** [comment]
That's cool. Can I use some of your code in my project?
As I understand the source code is here https://github.com/florintanasa/OpenExpertSDR right?

**Comment by florintanasa** [comment]
Yes, you can use what you wish because finally this is GPL idea.
The code from https://github.com/florintanasa/OpenExpertSDR is version before (27.04.2015) to add the changes with two audio cards support and version from copy.com is the last your code.
In https://github.com/florintanasa/Plugins you have SunSDR pkugins also ported but sCtrl is not ported in this moment.
I have in plan to use OpenExpertSDR toghether with teensy 2 https://github.com/florintanasa/PanelG11 to control my  SDR.
What I can say is that, qt5 is more better qt4 because version for qt5 use less, in linux, cpu (approx. a half) in comparation with qt4 version.
At Robert you found also DttSP lib ported to linux https://github.com/wd8rde/libDttSP
Also at my youtube channel you found some test https://www.youtube.com/channel/UCfwSU8kkGVOubW2RYaPqX-g
I am not a programmer but I like this software OpenExpertSDR and this make me to learn c++ and to start to ported to linux and finally to make, if is possible, to work on arm embedded device  Odroid U2/U3 but for this is necessary pnarama.cpp to port to OpenGL ES.
You make a great work at OpenExpertSDR.

73, YO8TFF, Florin Tanasa
